benchmark/README.md

```diff
@@ -0,0 +1,8 @@
+How to run these benchmarks:
+```
+GOOS=linux GOARCH=amd64 CC=clang CGO_CFLAGS="-I /usr/vagrant/repos/tracee/tracee-ebpf/dist/libbpf/usr/include" CGO_LDFLAGS="/usr/vagrant/repos/tracee/tracee-ebpf/dist/libbpf/libbpf.a" go test -exec 'sudo -E' -v -bench=. -benchtime=1x -tags=opa_wasm 
```

Comment:
Perhaps we should point this to a libbpf submodule in the repo (we have one in `tracee-ebpf/3rdparty/libbpf`) where we can find the headers and build libbpf.a